index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/6d267a36ba.js" crossorigin="anonymous"></script>
+    <title>Portfolio Template</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="main">
+        <div>
+            <nav>
+                <ul class="menu">
+                    <li>Portfolio</li>
+                    <li>About</li>
+                    <li>Contact</li>
+                </ul>
+            </nav>
+            <div class="h-menu">
+                <h1>Welcome</h1>
```

Comment:
- [ ] The use of semantic tags is imperative but it must be used with purpose. In this case, a more useful tag would have been a `a` tag because this is more like your `logo` and it should always redirect users back to the homepage for good user experience. Moreso, it becomes confusing for Search Engines and other accessibility devices when you have more than 1 `h1` in your code. Kindly use an anchor - `a` tagfor this please :pray: